recipe/meta.yaml

```diff
@@ -36,41 +39,53 @@ requirements:
     - make  # [unix]
     - gnuconfig
     - ninja
-    # To have the protoc compiler as a build tool.
-    - protobuf
-    # To have the thrift compiler as a build tool.
-    - thrift-cpp
+    - patch    # [unix]
+    - m2-patch # [win] 
   host:
-    - abseil-cpp 20211102.0  # [not osx]
+    - libabseil 20240116.2
     - aws-sdk-cpp 1.10.55
     - aws-crt-cpp 0.18.16
-    - boost-cpp 1.82
+    - boost-cpp {{ boost }}
     - brotli 1.0.9
-    - bzip2 1.0.8
+    - bzip2 {{ bzip2 }}
     # Only required and used on Windows.
-    - c-ares 1.18.1  # [win]
+    - c-ares 1.19.1  # [win]
+    - gflags 2.2.2
     - glog 0.5.0
-    - grpc-cpp 1.48.2
+    - libgrpc 1.62.2
     - lz4-c 1.9.4
     - openssl {{ openssl }}
-    - orc 1.7.4
-    - libprotobuf {{ libprotobuf }}
```

Comment:
is it possible to merge the aggregate branch i.e. the protobuf config change and keep this the same?

Reply:
I find it's still early in the in the rebuild effort.